icu4c/source/config/dist.mk

```diff
@@ -25,15 +25,16 @@ DISTY_IN=$(DISTY_DATA)/in
 DOCZIP=icu-docs.zip
 
 SVNTOP=$(top_srcdir)/..
-SVNVER=$(shell svnversion $(SVNTOP) | cut -d: -f1 | tr -cd 'a-zA-Z0-9')
-SVNURL=$(shell svn info $(SVNTOP) | grep '^URL:' | cut -d: -f2-)
+GITVER=$(shell (cd $(SVNTOP) && git describe --tags --exact-match || git rev-parse --short HEAD) 2>/dev/null || echo 'unknown')
+#GITBRANCH=$(shell (cd $(SVNTOP) && git describe --tags --exact-match  || git symbolic-ref -q --short HEAD) 2>/dev/null || echo unknown)
+SVNVER=$(GITVER)
```

Comment:
I think it would make more sense to instead update the rest of the file to use `GITVER` instead of `SVNVER`.